packages/lab/src/tabs-next/TabstripNext.css

```diff
@@ -7,8 +7,6 @@
   position: relative;
   background: transparent;
   width: 100%;
-  padding-left: var(--salt-spacing-300);
```

Comment:
This change will remove the padding from the "main" variant of the tabs too where it's needed. Could you update this please